build_web_compilers/lib/src/web_entrypoint_builder.dart

```diff
@@ -7,14 +7,64 @@ import 'dart:async';
 import 'package:analyzer/analyzer.dart';
 import 'package:build/build.dart';
 
+import 'dart2js_bootstrap.dart';
 import 'dev_compiler_bootstrap.dart';
 
 const ddcBootstrapExtension = '.dart.bootstrap.js';
 const jsEntrypointExtension = '.dart.js';
 const jsEntrypointSourceMapExtension = '.dart.js.map';
 
+/// Which compiler to use when compiling web entrypoints.
+enum WebCompiler {
+  Dart2Js,
+  DartDevc,
+}
+
+/// The top level keys supported for the `options` config for the
+/// [WebEntrypointBuilder].
+const _supportedOptions = const ['compiler', 'dart2js_args'];
+
+/// A builder which compiles entrypoints for the web.
+///
+/// Supports `dart2js` and `dartdevc` today.
```

Comment:
[nit] 'today' shouldn't be necessary to specify (unless we have short term plans to change it)

Reply:
done